scripts/test-maven-generate-jacoco-report.bash

```diff
@@ -46,7 +46,10 @@ if [ -z "$M2_LOCATION" ]; then
 	M2_LOCATION="./.m2"
 fi
 
+#./mvnw jacoco:report \
+#	--no-transfer-progress \
+#	-Dmaven.repo.local="${M2_LOCATION}" \
+#	--offline
```

Comment:
Is there a reason for the commented out code?

Reply:
Removed now,